error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/OptionalRulesTestInput.java

```diff
@@ -119,4 +119,12 @@ ImmutableSet<Optional<String>> testOptionalFilter() {
   Optional<String> testOptionalMap() {
     return Optional.of(1).stream().map(String::valueOf).findAny();
   }
+
+  ImmutableSet<String> testOptionalOrElseGet() {
+    return ImmutableSet.of(
+        Optional.of("foo").orElse("bar"),
```

Comment:
We make sure to use distinct values for every statement :).

Reply:
Does it need to be distinct for mutation testing?

Reply:
Nope, it's about making sure that Refaster properly moves things around (which is easier to see if all values are distinct). Unfortunately for technical reasons our Refaster tests aren't exercised by Pitest.